app/models/link.rb

```diff
@@ -1454,6 +1456,18 @@ def quantity_enabled_state_is_allowed
       end
     end
 
+    def isbn_format_is_valid
+      return if isbn.blank?
+
+      # Remove hyphens and spaces for validation
+      normalized_isbn = isbn.to_s.gsub(/[-\s]/, "")
+
+      # ISBN-13 format: 13 digits
+      unless normalized_isbn.match?(/\A\d{13}\z/)
+        errors.add(:isbn, "must be a valid ISBN-13 (13 digits, optionally with hyphens)")
+      end
```

Comment:
Two issues:
- The checksum validation is missing
- ISBN-10 format validation is missing